2-TokenCache/Console-TokenCache/Console-TokenCache.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp3.1; net472</TargetFrameworks>
```

Comment:
Is there a purpose for supporting 2 frameworks ? I would keep it simple and focus on netcoreapp only.

Reply:
That was my recommendation.

Reply:
Just wanted to understand why ? Will there be an option to enable / disable the embedded webview on .net classic?

Reply:
That's a good idea, actually. @TiagoBrenck: do we want to add a menu item for this?

Reply:
@bgavrilMS could you help me with this embedded webview topic for .net classic please? I didnt get your point.

Reply:
You can add another menu item "Toggle embedded / system webview (current: system)". 
Depeding on which one is selected, when performing the `AcquireTokenInteractive` call, you just configure it with `UseEmbeddedWebview(true/false)`

Note that embedded webview is not available on .netcore, so this will throw an exception, which is ok.

Reply:
@bgavrilMS  I ran a test and the embedded webview is true by default when I use .net classic. Would it still make sense to have the suggested menu item?

Reply:
Yes, there are advanges to using the system browser, such as WAM integration. So it is useful even for .net classic. 
In the future we will have an embedded browser solution for .net core 3 as well.

Reply:
Added menu option for embedded view